ui/org.eclipse.pde.core/src/org/eclipse/pde/internal/core/builders/JarManifestErrorReporter.java

```diff
@@ -139,17 +141,32 @@ protected void parseManifest(IDocument document, IProgressMonitor monitor) {
 				}
 				// parse
 				if (line.length() == 0) {
-					// Empty Line
+					// Empty Line in the beginning
 					if (l == 0) {
 						report(PDECoreMessages.BundleErrorReporter_noMainSection, 1, CompilerFlags.ERROR, PDEMarkerFactory.CAT_FATAL);
 						return;
+
+						// empty line elsewhere
+					} else if (l != document.getNumberOfLines() - 1) {
+						emptyLines.add(Integer.valueOf(l));
+						continue;
 					}
 					/* flush last line */
 					if (header != null) {
 						fHeaders.put(header.getName().toLowerCase(), header);
 						header = null;
 					}
 					break; /* done processing main attributes */
+				} else {
```

Comment:
This `else` is unneeded and shows whole block as aa huge warning in Eclipse:
![image](https://user-images.githubusercontent.com/964108/233018351-2d9f4c91-4082-443a-8038-542b40cdd123.png)

Reply:
fixed it, the redundant else block is taken out.